templates/zerver/emails/email_base.html

```diff
@@ -1,17 +1,350 @@
+        PRESERVE THESE STYLES IN THE HEAD
+        ------------------------------------- */
+
+        @media all {
+            .ExternalClass {
+                width: 100%;
+            }
+            .ExternalClass,
+            .ExternalClass p,
+            .ExternalClass span,
+            .ExternalClass font,
+            .ExternalClass td,
+            .ExternalClass div {
+                line-height: 100%;
+            }
+            .apple-link a {
+                color: inherit !important;
+                font-family: inherit !important;
+                font-size: inherit !important;
+                font-weight: inherit !important;
+                line-height: inherit !important;
+                text-decoration: none !important;
+            }
+        }
+        </style>
     </head>
-    <body>
-        <table width="80%" style="align:center; max-width:800px" align="center">
+    <body class="">
+        <table border="0" cellpadding="0" cellspacing="0" class="body">
             <tr>
-                <td style="font-size:16px; font-family:Helvetica;">
-                    {% block content %}{% endblock %}
+                <td>&nbsp;</td>
+                <td class="container">
+                    <div class="content">
+                        <div class="graphic-wrapper">
+                            {% block illustration %}{% endblock %}
+                        </div>
+                        <!-- START CENTERED WHITE CONTAINER -->
+                        <span class="preheader">This is preheader text. Some clients will show this text as a preview.</span>
+                        <table class="main">
+                            <!-- START MAIN CONTENT AREA -->
+                            <tr>
+                                <td class="wrapper">
+                                    <table border="0" cellpadding="0" cellspacing="0">
+                                        <tr>
+                                            <td>
+                                                {% block content %}{% endblock %}
+                                            </td>
+                                        </tr>
+                                    </table>
+                                </td>
+                            </tr>
+                            {% block manage_preferences %}{% endblock %}
+                            <!-- END MAIN CONTENT AREA -->
+                        </table>
+                        <!-- START FOOTER -->
+                        <div class="footer">
+                            <table border="0" cellpadding="0" cellspacing="0">
+                                <tr>
+                                    <td class="content-block">
+                                        <span class="apple-link">Kandra Labs, 112 S Park St, San Francisco CA 94107</span>
+                                        <br/> Don't like these emails? <a href="#">Unsubscribe</a>.
```

Comment:
it would make sense to make use of `manage_preferences` as it is adequately put in the followup emails, i completely agree. just having some trouble!